packages/volto/src/components/manage/Widgets/SelectStyling.jsx

```diff
@@ -112,10 +112,13 @@ export const customSelectStyles = {
   control: (styles, state) => ({
     ...styles,
     border: 'none',
-    borderBottom: '1px solid #c7d5d8',
-    boxShadow: 'none',
-    borderBottomStyle: state.menuIsOpen ? 'dotted' : 'solid',
```

Comment:
` if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.event.pull_request.base.repo.full_name
    runs-on: ubuntu-latest
    name: Core Volto Slate
    timeout-minutes: 45
`******_throw.in-0_`